src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File mainFile = new File(fromFileName);
+        String[] words = readFromFile(mainFile).split("\\W+");
```

Comment:
I think you should separate the [constant](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#use-constants).